### PR TITLE
feat: Database-setup

### PR DESCRIPTION
**Pull request by AbonyiXavier**
## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by neymarjimoh — CHANGES_REQUESTED** [review]
use double quotes

File: server/app.js [hunk 1]
```diff
@@ -38,4 +43,12 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+const PORT = process.env.PORT || 500;
```
Comment (Kibetchirchir) [inline server/app.js]: Use 5000 instead. Since 500 is already set and not available

File: server/app.js [hunk 2]
```diff
@@ -38,4 +43,12 @@ app.use(function(err, req, res, next) {
...
+
+db.sequelize.sync().then(() => {
+  app.listen(PORT, () => {
+    console.log(`Server listening at: http://localhost:${PORT}`);
+  });
+});
```
Comment (Kibetchirchir) [inline server/app.js]: use a different file to setup the DB and connection

File: server/config/config.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "dialect": "mysql",
+    "operatorsAliases": false
+  },
+  "production": {
+    "username": "root",
+    "password": null,
+    "database": "database_production",
+    "host": "127.0.0.1",
+    "dialect": "mysql",
+    "operatorsAliases": false
```
Comment (Kibetchirchir) [inline server/config/config.json]: Use `dotenv` to store sensitive data that can be used to exploit your app
check `dotenv` np package

**Review by Kibetchirchir — CHANGES_REQUESTED** [review]

**Review by sadiqasg — DISMISSED** [review]
This looks good @AbonyiXavier ,
Good call with adding babel

**Review by neymarjimoh — DISMISSED** [review]
Nice idea adding Babel.

**Review by sadiqasg — APPROVED** [review]
Looking good

**Pull request merged**